### PR TITLE
Updates Britecharts version

### PR DESCRIPTION
**Pull request by Golodhros**
## Description

Updates the version of Britecharts to 2.6.0
Updates all chart's proptypes
Removes deprecated validateData helper

## Motivation and Context

Update and fix of docs bug
Fixes: https://github.com/eventbrite/britecharts-react/issues/107

## How Has This Been Tested?

Ran tests

## Types of changes

- [ ] Refactor (changes the way we code something without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/charts/legend/LegendChart.js [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-    } else {
-        container.call(chart);
-    }
-
-    return chart;
-};
-
-legend.destroy = () => {};
-
-export default legend;
```
Comment (Golodhros) [inline src/charts/legend/LegendChart.js]: Something weird here with the "L" in the name, don't worry, below is the real one.

**Comment by dalerasrorov-eb** [comment]
Could we include new props to `sparkline` from this PR ?https://github.com/eventbrite/britecharts/commit/a21db1dddb5814ff663bd379cc9cc4e4e2572019

**Comment by Golodhros** [comment]
Sure! Checking those now!

File: src/charts/donut/Donut.js [hunk 2]
```diff
@@ -11,11 +11,21 @@ export default class Donut extends Component {
...
+
         /**
          * Gets or Sets the colorSchema of the chart
          */
         colorSchema: PropTypes.arrayOf(PropTypes.string),
 
+        /**
+         * Gets or Sets the emptyDataConfig of the chart. If set and data is empty (quantity adds up to zero or there are no entries), the chart will render an empty slice with a given color (light gray by default)
+         */
+        emptyDataConfig: PropTypes.object,
```
Comment (dalerasrorov-eb) [inline src/charts/donut/Donut.js]: After `2.6.0` this should be switched to `emptySliceColor`: https://github.com/eventbrite/britecharts/pull/539

**Pull request merged**